doc/source/debug.rst

```diff
@@ -0,0 +1,188 @@
+Debugging Kubernetes
+====================
+
+Sometimes your Kubernetes deployment doesn't behave the way you'd expect.
+This section provides some tips on debugging and fixing some common problems.
+
+Debugging commands
+------------------
+In order to debug your Kubernetes deployment, you need to be able to inspect
+the state of the pods being used. The following are a few common commands
+for debugging.
+
+.. note::
+
+   You may need to add ``--namespace=<YOUR_NAMESPACE>`` in order for the

```

Comment:
You always need to, and I think we should put this in every command. It won't work without it.